### PR TITLE
Onboarding: update screenshots

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Onboarding: update screenshots
- add step on analytics
- Reformat procedure to use numbered steps / procedural format

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/getting-started/onboarding.markdown [hunk 1]
```diff
@@ -5,18 +5,41 @@ description: "Instructions to get Home Assistant configured."
...
+    ![Define your location specific settings.](/images/getting-started/onboarding_location.png)
+    
+1. Select which information you are willing to share. 
+    * Sharing is disabled by default. However, we would like to encourage you to share some of this data. 
+    * This information helps us to find out which platforms we need to support and where to focus our efforts.
+    * The data is anonymized and aggregated. To see the charts we generate out of this data, take a look at our [analytics page](https://analytics.home-assistant.io/). 
+    
+   ![Share anonymized data](/images/getting-started/onboarding_share_anonymized_info.png) 
+
+1. Once you are done, click **Next**.
```
Comment (c0ffeeca7) [inline source/getting-started/onboarding.markdown]:
```suggestion
1. Once you are done, select **Next**. 
```

File: source/getting-started/onboarding.markdown [hunk 2]
```diff
@@ -5,18 +5,41 @@ description: "Instructions to get Home Assistant configured."
...
+    
+   ![Share anonymized data](/images/getting-started/onboarding_share_anonymized_info.png) 
+
+1. Once you are done, click **Next**. 
+    * Home Assistant will then show any {% term devices %} that it has discovered on your network. 
+    * Don't be alarmed if you see fewer items than shown below; you can always manually add devices later.
+
+    ![Discovery of devices on your network.](/images/getting-started/onboarding_devices.png)
+
+1. Finally, click **Finish**.
```
Comment (c0ffeeca7) [inline source/getting-started/onboarding.markdown]:
```suggestion
1. Finally, select **Finish**. 
```

**Review by c0ffeeca7 — COMMENTED** [review]
style guide

File: source/getting-started/onboarding.markdown [hunk 3]
```diff
@@ -5,18 +5,41 @@ description: "Instructions to get Home Assistant configured."
 
 Alright, you made it here. The tough part is done.
 
-With Home Assistant installed, it's time to configure it. Here you will create the owner account of Home Assistant. This account will be an administrator and will always be able to change everything. Enter a name, username, password and select **create account**.
+After entering the Home Assistant devices address in the address bar of your browser, the preparation screen is shown. Depending on your hardware, preparation may take a while.
```
Comment (frenck) [inline source/getting-started/onboarding.markdown]:
```suggestion
After entering the Home Assistant device's address in your browser's address bar, the preparation screen is shown. Depending on your hardware, preparation may take a while.
```

File: source/getting-started/onboarding.markdown [hunk 4]
```diff
@@ -5,18 +5,41 @@ description: "Instructions to get Home Assistant configured."
...
+    
+1. Select which information you are willing to share. 
+    * Sharing is disabled by default. However, we would like to encourage you to share some of this data. 
+    * This information helps us to find out which platforms we need to support and where to focus our efforts.
+    * The data is anonymized and aggregated. To see the charts we generate out of this data, take a look at our [analytics page](https://analytics.home-assistant.io/). 
+    
+   ![Share anonymized data](/images/getting-started/onboarding_share_anonymized_info.png) 
+
+1. Once you are done, select **Next**. 
+    * Home Assistant will then show any {% term devices %} that it has discovered on your network.
```
Comment (frenck) [inline source/getting-started/onboarding.markdown]:
```suggestion
    * Home Assistant will then show any {% term devices %} it has discovered on your network.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
LGTM, feel free to merge when it is ready 👍 

../Frenck

**Comment by c0ffeeca7** [comment]
thank you!

**Pull request merged**